docs/src/appendix/cast/account/create.md

```diff
@@ -0,0 +1,24 @@
+# `create`
+Create all data required for account deployment.
+Account information will be saved inside `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` or in the file
+specified with `--accounts-file` argument.
+
+## Required common arguments - passed by value or specified in Scarb.toml
+
+* [`network`](../common.md#--network--n-network_name)
+* [`url`](../common.md#--url--u-rpc_url)
+
+## `--name, -n <ACCOUNT_NAME>`
+Required.
+
+Account name under which account information is going to be saved.
+
+## `--salt, -s <SALT>`
+Optional.
+
+Salt for the account address. If omitted random one will be used.
+
+## `--add-profile, -a`
+Optional.
+
+If passed, a profile with corresponding data will be created in Scarb.toml
```

Comment:
```suggestion
If passed, a profile with corresponding data will be added in Scarb.toml
```

Reply:
btw, will that include rpc url? Should it?

Reply:
Yes it would and I think it also should

Reply:
I see. technically, the same account would be available through any rpc url as long as it's deployed right? But I guess it maybe makes sense to add this url as well, since users probab;y won't switch between urls that much

Reply:
Yup, you're right